js/helpers.js

```diff
@@ -69,6 +69,23 @@ define( [ "jquery", "./ns", "jquery-ui/jquery.ui.core" ], function( jQuery ) {
 		// Place to store various widget extensions
 		behaviors: {},
 
+		// Direct focus to the page title, or otherwise first focusable element
+		focusPage: function( page ) {
+			var autofocus = page.find( "[autofocus]" ),
+				pageTitle = page.find( ".ui-title:eq(0)" );
```

Comment:
lets just set the var here, and do the `find` later don't need to needlessly search the whole page if we already found an `autofocus` element.